packages/components/src/components/link-social/link-social.tsx

```diff
@@ -51,11 +52,23 @@ export class LinkSocial {
 
   public componentWillLoad(): void {
     throwIfInvalidLinkUsage(this.host, this.href);
+    console.warn(
```

Comment:
Iam not sure, but if there are more than one component that get deprecated id could make sense to create a helper for this and reuse to ensure the warning is everywhere the same?

Reply:
Helper function was created.